ivy/func_wrapper.py

```diff
@@ -1101,16 +1101,8 @@ def method(*args, **kwargs):
             set(ivy.invalid_dtypes)
```

Comment:
This should also be updated to `ivy.function_unsupported_devices_and_dtypes` instead of `ivy.function_unsupported_dtypes` right?

Reply:
yep